README.md

```diff
@@ -55,6 +55,17 @@ For example:
 katex.render("c = \\pm\\sqrt{a^2 + b^2}", element, { displayMode: true });
 ```
 
+Note that the `throwOnError` option only deals with how TeX related parsing errors are handled.
```

Comment:
Depends on whether we want to keep this paragraph at all. If so, then yes, this would be better. “missing” is still inferior to “unsupported”, because it could also mean commands missing from the input like e.g. a missing `\right` or `\end`. If we drop this paragraph, then I'd not include the “such as mismatched braces” portion in order to keep the bullet list concise.